packages/angular/cli/package.json

```diff
@@ -43,6 +43,12 @@
     "uuid": "^3.3.2"
   },
   "ng-update": {
-    "migrations": "@schematics/angular/migrations/migration-collection.json"
+    "migrations": "@schematics/angular/migrations/migration-collection.json",
+    "packageGroup": {
+      "@angular/cli": "0.0.0",
+      "@angular-devkit/build-angular":  "0.0.0",
+      "@angular-devkit/build-ng-packagr": "0.0.0",
+      "@angular-devkit/build-webpack": "0.0.0"
```

Comment:
Is this needed?

Isn’t this a transitive dependency of build-angular?

Reply:
Could be used directly as well.  If not present here the 7.x version would still be left after an upgrade (and non-functional) in that scenario.